command.go

```diff
@@ -930,6 +934,23 @@ func (c *Command) execute(a []string) (err error) {
 		argWoFlags = a
 	}
 
+	if c.OnKillRun != nil {
+		sigchan := make(chan os.Signal)
+		signal.Notify(
+			sigchan,
+			syscall.SIGINT,
+			syscall.SIGKILL,
```

Comment:
By definition, at least on POSIX, SIGKILL cannot be caught.  I've tested it on my Mac and I can confirm an `OnKillRun` function does not get called.  So I suggest removing this one.

In fact, the linter reports it should be removed 😄